src/plugins/npm/index.ts

```diff
@@ -1,3 +1,5 @@
+import setToken from '@hutson/set-npm-auth-token-for-ci';
+import envCi from 'env-ci';
```

Comment:
Since you're only using the `isCi` function, I would recommend [is-ci](https://www.npmjs.com/package/is-ci) instead. It's [~17kb](https://packagephobia.now.sh/result?p=is-ci) instead of [~250kb](https://packagephobia.now.sh/result?p=env-ci).

Reply:
I originally had that but after this I changed it https://github.com/intuit/auto-release/pull/146#issuecomment-451767743

I'll take this chance to change it back since we currently do not use any of the other functions. If we need that functionality in the future it should be simple to add.

Reply:
changed!